pipeline/metadata/qaqc_table.csv

```diff
@@ -79,6 +79,14 @@ sonde-turbidity,1 day,,threshold=8,1 hour,
 sonde-vpos,1 day,,threshold=8,1 hour,
 sonde-wipercur,1 month,,threshold=8,1 hour,
 sonde-wiperpos,,,,1 hour,Too variable; no QAQC
+sonde-chlorophyl,,,,,
+sonde-chl-rfu,,,,,
+sonde-chl-raw,,,,,
+sonde-chl-cells,,,,,
+sonde-rhodamine,,,,,
```

Comment:
Nothing wrong here, just curious. Will the pipeline stop of a research name is found in test data but doesn't have a row in this qaqc file?

Reply:
Yes, it will